app/views/devise/registrations/new.html.erb

```diff
@@ -1,31 +1,72 @@
 <h2>Sign Up</h2>
 
-<%= form_for(resource, :as => resource_name, :url => registration_path(resource_name)) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+<% content_for :stylesheets do %>
+  <%= stylesheet_link_tag "signup_form" %>
+<% end %>
 
+<% content_for :javascripts do %>
+  <%= javascript_include_tag "signup_form" %>
+<% end %>
+
+<%= form_for(resource, :as => resource_name, :url => registration_path(resource_name)) do |f| %>
 	<div class="input-field">
-	  <%= f.text_field :first_name, :autofocus => true %>
+		<%= f.text_field :first_name, :title => "", :pattern => ".+", :required => true, :autofocus => true %>
 		<%= f.label :first_name %>
+    <div class="field-hint">
```

Comment:
There seems to be mixing of tabs and spaces in this file. I know that the file was already like that, but could you switch indentations to spaces?